go/src/koding/kites/tunnelproxy/client.go

```diff
@@ -135,8 +136,18 @@ func NewClient(opts *ClientOptions) (*Client, error) {
 		optsCopy.Log = logging.NewCustom("tunnelclient", optsCopy.Debug)
 	}
 
+	// TODO(rjeczalik): fix production to use WebSocket by default
```

Comment:
Yes I know, it uses websocket for tunnelserver only. This TODO has no deadline :-)